src/webhooks/pubsub/stalebot.test.ts

```diff
@@ -74,12 +79,21 @@ But! If you comment or otherwise update it, I will reset the clock, and if you r
     ]);
   });
 
-  it('should close issue if there is no activity after a week and an issue is stale', async function () {
+  it('should not close issue if there is no activity after a week and an issue is stale but does not have label `Waiting for: Community`', async function () {
```

Comment:
holy triple negative, batman!

What about: "should not close stale issue that has been inactive for more than a week but does not have label `Waiting for: Community`"?